docs/gateway/s3.md

```diff
@@ -0,0 +1,51 @@
+# Minio S3 Gateway [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+Minio S3 Gateway adds Minio features like Minio Browser and Caching to AWS S3 buckets.
+
+## Run Minio Gateway for AWS S3
+
+As a pre-requisite to run Minio S3 gateway, you need valid AWS S3 access key and secret key.
+
+### Using Docker
+
+```
+docker run -p 9000:9000 --name minio-s3 \
+ -e "MINIO_ACCESS_KEY=aws_s3_access_key" \
+ -e "MINIO_SECRET_KEY=aws_s3_secret_key" \
+ minio/minio gateway s3
+```
+
+### Using Binary
+
+```
+export MINIO_ACCESS_KEY=aws_s3_access_key
+export MINIO_SECRET_KEY=aws_s3_secret_key
+minio gateway s3
+```
+
+## Minio Caching
+
+Minio caching allows storing content closer to the applications. Frequently accessed objects are stored in the cache. Minio caching with Minio gateway feature allows you to
+
+- Dramatic improvements for time to first byte for any object.
+- Avoid S3 [data transfer charges](https://aws.amazon.com/s3/pricing/).
+
+Refer [this document](https://docs.minio.io/docs/minio-disk-cache-guide.html) to get started with Minio Caching.
+
+## Minio Browser
+
+Minio Gateway comes with an embedded web based object browser. Point your web browser to http://127.0.0.1:9000 to ensure that your server has started successfully.
```

Comment:
The browser link will not be http://127.0.0.1:9000 in the case of a docker instance.

Reply:
since we are mapping ports in the example command `-p 9000:9000` browser will be accessible at http://127.0.0.1:9000

Reply:
ok